code-studio/scripts/build-js.js

```diff
@@ -39,6 +39,10 @@ Promise.all([
       'levels/contract_match.jsx',
       'levels/widget.js',
       'levels/external.js',
+      'levels/multi.js',
+      'levels/textMatch.js',
+      'levels/levelGroup.js',
+      'levels/dialogHelper.js',
```

Comment:
i suspect dialoghelper is used by a lot of other places.. are they all okay with this change?